tests/integration/test_cross_model_remote_write.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Comment:
Naming convention: we already have `test_remote_write_*`. Should this one be named `test_remote_write_cross_model` to match existing?